charts/retool/values.yaml

```diff
@@ -95,6 +95,9 @@ externalSecrets:
   # Support for legacy external secrets, note this is deprecated in favour of External Secrets Operator: https://github.com/godaddy/kubernetes-external-secrets
   # This mode only allows a single secret name to be provided.
   enabled: false
+  # If external secrets are currently enabled, it is disallowed to specify regular configuration secrets as a safeguard from clobbering.
+  # This flag allows bypassing that check and specifying both an ExternalSecret and a regular secret for different secrets.
+  allowOtherSecrets: false
```

Comment:
thinking naming...maybe `includeConfigSecrets`?

also, do we want to doc a recommendation towards true/false? is the `false` default what we'd usually want or just for backcompat?

Reply:
yup, good idea. and yeah false the default just for backcompat